packages/server/index.ts

```diff
@@ -1,20 +1,70 @@
 import dotenv from 'dotenv'
 import cors from 'cors'
 dotenv.config()
+import {
+  createEmoji,
+  getEmojisByMessageID,
+  deleteUserByIds,
+} from './models/emoji'
 
 import express from 'express'
-import { createClientAndConnect } from './db'
+// import { createClientAndConnect } from './db'
+import { dbConnect } from './init'
 
 const app = express()
 app.use(cors())
+app.use(express.json())
 const port = Number(process.env.SERVER_PORT) || 3001
 
-createClientAndConnect()
+// createClientAndConnect()
+dbConnect()
 
 app.get('/', (_, res) => {
   res.json('👋 Howdy from the server :)')
 })
 
+// ?messageID=1  query параметры
+app.get(`/api/emoji`, async (req, res) => {
+  const messageID = Number(req.query.messageID)
+  if (!messageID) res.status(400).send({ reason: 'bad request' })
+  const emojis = await getEmojisByMessageID({ messageID })
+  console.log('emojis', emojis)
+  if (emojis.length === 0)
+    res.status(400).send({ reason: 'emojis does not exist' })
+  res.send(emojis)
+})
+
+app.post(`/api/emoji`, async (req, res) => {
+  const { messageID, unicod, userID } = req.body
+  if (
+    !messageID ||
+    !unicod ||
+    !userID ||
+    typeof messageID !== 'number' ||
+    typeof unicod !== 'string' ||
+    typeof userID !== 'number'
```

Comment:
[Предложение]
Мне такую запись труднее читать, чем отрицание положительного условия. Я бы сделала отрицание всего условия с И:
```suggestion
    !(messageID &&
    unicod &&
    userID &&
    typeof messageID === 'number' &&
    typeof unicod === 'string' &&
    typeof userID === 'number')
```
А лучше вообще поменяла бы местами отправку ошибки и положительный результат. То есть здесь сделала бы положительное условие:
```
if (
    messageID &&
    unicod &&
    userID &&
    typeof messageID === 'number' &&
    typeof unicod === 'string' &&
    typeof userID === 'number'
) {
    const emoji = await createEmoji(req.body);

    if (emoji) {
        res.send(emoji);
        return;
    }
    res.status(400).send({ reason: 'emoji already exist' });
    return;
}

res.status(400).send({ reason: 'bad request' });
```

Reply:
Понял, посмотрю, спасибо!